### PR TITLE
Feature/player-activity-drawer

### LINKED ISSUE #59: feature/player-activity-drawer

### PR DESCRIPTION
**Pull request by zaktolba**

### CODE REVIEW

File: src/components/Party/Drawer/ActivityDrawerWidget.vue [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+                }
+            }
+        }
+
+        if (interval > 1 || interval === 0) {
+            intervalType += 's';
+        }
+
+        return interval + ' ' + intervalType;
+    },
```
Comment (MrZyr0) [inline src/components/Party/Drawer/ActivityDrawerWidget.vue:73]: Omg qu'est-ce-que c'est que cette fonction ? 😱

File: src/components/Party/Drawer/ActivityDrawerWidget.vue [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+.drawer * {
+  pointer-events: auto;
+}
+
+.drawer > input[type=checkbox] {
+	display: none;
+}
+
+.drawerPull {
+	display: inline-block;
```
Comment (Laroche-Nicolas) [inline src/components/Party/Drawer/ActivityDrawerWidget.vue:124]: Maybe just fix indentation

File: src/components/Party/Drawer/ActivityDrawerWidget.vue [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+        return actionType;
+    }
+  },
+  computed: {
+    ...mapGetters(['user']),
+  }
+}
+</script>
+
+<style scoped>
```
Comment (Laroche-Nicolas) [inline src/components/Party/Drawer/ActivityDrawerWidget.vue:96]: Maybe fix indentation in CSS

**Review by Laroche-Nicolas — CHANGES_REQUESTED** [review]

File: src/components/Party/Drawer/ActivityDrawerWidget.vue (hunk 3 above)
Comment (zaktolba) [reply src/components/Party/Drawer/ActivityDrawerWidget.vue:96]: already like that in all the vue components

**Review by Laroche-Nicolas — APPROVED** [review]

**Review by MrZyr0 — APPROVED** [review]

File: src/components/Party/Drawer/ActivityDrawerWidget.vue [hunk 4]
```diff
@@ -0,0 +1,230 @@
...
+<script>
+import {mapGetters} from 'vuex'
+
+export default {
+  name: "StatDrawerWidget",
+  props: {
+    actions: { type: Array },
+  },
+  methods: {
+    timeSince(date) {
```
Comment (MaxenceMottard) [inline src/components/Party/Drawer/ActivityDrawerWidget.vue:31]: Next time you can use moment js, i love this library.

Comment (zaktolba) [reply src/components/Party/Drawer/ActivityDrawerWidget.vue:31]: Noted, good to know because I hesitated using it

**Pull request merged**